gcp_variant_transforms/testing/integration/presubmit_tests/small_tests/infer_undefined_header_fields.json

```diff
@@ -0,0 +1,53 @@
+{
+  "test_name": "infer-undefined-header-fields",
+  "table_name": "infer_undefined_header_fields",
+  "input_pattern": "gs://gcp-variant-transforms-testfiles/small_tests/infer-undefined-header-fields.vcf",
+  "infer_undefined_headers": "True",
+  "runner": "DataflowRunner",
+  "assertion_configs": [
+    {
+      "query": ["NUM_ROWS_QUERY"],
+      "expected_result": {"num_rows": 1}
+    },
+    {
+      "query": ["SELECT COUNT(AA) AS cnt FROM {TABLE_NAME}"],
+      "expected_result": {"cnt": 1}
+    },
+    {
+      "query": ["SELECT COUNT(AF) AS cnt FROM {TABLE_NAME}"],
+      "expected_result": {"cnt": 1}
+    },
+    {
+      "query": ["SELECT COUNT(DB) AS cnt FROM {TABLE_NAME}"],
+      "expected_result": {"cnt": 1}
+    },
+    {
+      "query": ["SELECT COUNT(DP) AS cnt FROM {TABLE_NAME}"],
+      "expected_result": {"cnt": 1}
+    },
+    {
+      "query": ["SELECT COUNT(H2) AS cnt FROM {TABLE_NAME}"],
+      "expected_result": {"cnt": 1}
+    },
+    {
+      "query": ["SELECT COUNT(NS) AS cnt FROM {TABLE_NAME}"],
+      "expected_result": {"cnt": 1}
+    },
+    {
+      "query": ["SELECT SUM(HQ) AS sum FROM {TABLE_NAME} t, t.call as call,",
```

Comment:
nit: please use this indendation format to be consistent with the rest of the [cases](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/testing/integration/presubmit_tests/medium_tests/test_1000_copies_of_valid_4_2_with_merge.json#L45):
```
{
  "query": [
    "SELECT SUM(HQ) AS sum FROM {TABLE_NAME} t, t.call as call,",
    "call.HQ as HQ"
  ],
  "expected_result": {"sum": 204}
}
```

Reply:
Done!